GPU/Common/TextureCacheCommon.cpp

```diff
@@ -1866,7 +1866,7 @@ bool CanDepalettize(GETextureFormat texFormat, GEBufferFormat bufferFormat) {
 // Great enhancement for Test Drive.
 static bool CanUseSmoothDepal(const GPUgstate &gstate, GEBufferFormat framebufferFormat, int rampLength) {
 	if (gstate.getClutIndexStartPos() == 0 &&
-		gstate.getClutIndexMask() <= rampLength) {
+		gstate.getClutIndexMask() < rampLength) {
```

Comment:
What if all 256 entries are a ramp?  Then we'll count up to 255.

-[Unknown]

Reply:
Right .. The goal is to make sure that one past the mask is part of the ramp (so that filtering doesn't pull in information from the next entry), but when it's the full palette, that's not relevant due to clamp. So that'll need some extra logic.

Reply:
Oh yeah, an additional point - with 256 entries, smoothing is equivalent to non-smoothing since our framebuffers don't have more precision than 8 bits per channel...

Reply:
Oh, then I guess it doesn't matter anyway, you're right.

-[Unknown]